### PR TITLE
Prettier eslint feature

### PR DESCRIPTION
**Pull request by drojas9764**
Added eslint and prettier to manage code

### CODE REVIEW

**Comment by vshypivdychgonewfirecom** [comment]
Looks good, but I don't see git hooks  
Something like this one 
https://prettier.io/docs/en/precommit.html

File: 6 [hunk 1]
```diff
@@ -0,0 +1,12 @@
+
```
Comment (vshypivdychgonewfirecom) [inline 6:1]: Didn't get why name is **6**  
Is it ok ?

File: .prettierrc [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+	"useTabs": true,
```
Comment (vshypivdychgonewfirecom) [inline .prettierrc:2]: @MateusKawazoe-ViaConsulting  Areyou agree with this formating ?

Comment (Mateuskwz-NewFire) [reply .prettierrc:2]: Sure

**Pull request merged**